### ISSUE TITLE
Add opportunity to make “C” label visible

### ISSUE THREAD

**Issue by lizavr** [epam/ketcher#5106]
As a User, I want to have an opportunity to control “C” label visibility if the atom is a carbon. Now, it's displayed only if angle is 180 deg.

**Solution**
As a solution it can be additional property for atom to set up label visibility in .ket format. (Currently model doesn't have any flags for this)

![image](https://github.com/user-attachments/assets/ee18cde4-1141-40b4-a0b3-6538c9bae377)

[ket.txt](https://github.com/user-attachments/files/16233191/ket.txt)

**Comment by flange-ipb** [comment]
You can achieve this already with the setting "Display carbon explicitly" (under Settings -> Atoms). This option can also be set programmatically via `ketcher.editor.options({"carbonExplicitly":true})`.

**Comment by lizavr** [comment]
Yes, but however this applies to all atoms in the molecule. If I want to specify it for just one atom, I can't do that. Or perhaps there's a way to set this just for specific atom?

**Comment by flange-ipb** [comment]
Alight, then an atom alias might be an option. You can find the "Alias" field by double-clicking an atom (see screenshot).
![Screenshot 2024-07-16 at 15-32-58 Ketcher v2 21 0](https://github.com/user-attachments/assets/2041a8fc-8d12-47b7-8b90-f0db6bfbf60e)

In the Ket format this boils down to the `alias` field of an atom object:
```
{
    "label": "C",
    "alias": "C",
    "location": [
        17.157106781186542,
        -8.967893218813455,
        0
    ]
}
```

**Comment by lizavr** [comment]
Thank you!
